rotkehlchen/exchanges/kraken.py

```diff
@@ -153,7 +153,14 @@ def history_event_from_kraken(
                 else:
                     event_subtype = HistoryEventSubType.RECEIVE
             elif event_type == HistoryEventType.STAKING:
-                event_subtype = HistoryEventSubType.REWARD
+                # in the case of ETH.S after the merge rewards don't compound unlike what happens
+                # with other assets and a virtual event with negative amount is created
+                if asset == A_ETH2 and raw_amount < ZERO:
+                    event_type = HistoryEventType.INFORMATIONAL
+                    notes = 'Automatic virtual conversion of staked ETH rewards to ETH'
+                    raw_amount = AssetAmount(abs(raw_amount))
```

Comment:
nitpick: It could be `raw_amount = AssetAmount(-raw_amount)` to be a bit more clear